core/src/test/java/org/ehcache/EhcacheBasicGetTest.java

```diff
@@ -148,7 +150,8 @@ public void testGetNoStoreEntryCacheLoadingException() throws Exception {
     try {
       ehcache.get("key");
       fail();
-    } catch (CacheLoadingException e) {
+    } catch (CachePassThroughException e) {
```

Comment:
Ehcache should not leak `CachePassThroughException`, this should still work by catching the `CacheLoadingException`